.bumpversion.cfg

```
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 2.0.0
+current_version = 2.1.0
```

This is going to be a major version bump, backwards incompatible, due to the changes to the variable inputs.

```suggestion
current_version = 3.0.0
```

---

Updated bump > 3.0.0